### DISCUSSION TITLE
How stable is the current beta of V3?

### DISCUSSION THREAD

**Discussion by yasin-05** [rappasoft/laravel-livewire-tables#1502] [General]
Hi,

So I'm in a situation where I need to upgrade some packages to work with L10. However, these packages require Livewire v3 as a dependency. 

As you can imagine this would mean I also need to upgrade rappasoft/laravel-livewire-tables to the beta version for v3 support. I want to ask how stable is the current beta of this package? Is there many bugs still pending to be resolved, or is there an estimate of when a stable version will be released?

Lastly, big thanks to the author, and contributors for their time, and efforts in development of this package. Much appreciated.

Thanks

**Comment by lrljoe** [comment]
So it's now stable at v3.1.0 and officially released as such.
